syncopy/io/_load_nwb.py

```diff
@@ -106,8 +108,6 @@ def load_nwb(filename, memuse=3000):
             channels = acqValue.electrodes[:].location
```

Comment:
If I understand correctly, you create separate `AnalogData` objects for each `acquisition` in the NWB file (which makes a lot of sense!). In that case I'm not sure, if the existing channel counting logic is then still valid: https://github.com/esi-neuroscience/syncopy/blob/d5a735cb6bf9c3ad07f88c85bf0b2ef2957253f7/syncopy/io/_load_nwb.py#L118

Reply:
Ah sorry, nChannels is unused now. We can remove it

Reply:
Ah, of course - sorry, didn't see that. Yes I guess removing it avoids confusion.